src/backer/constants.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+SAID = os.environ.get('SAID') if 'SAID' in os.environ else None
```

Comment:
`REGISTRAR_SEAL_SAID` would be clearer. Can we also add this to the env.example and add the one we calculated for the preview network there?

Reply:
updated
ref here: 9341a6c17b6fac709ccee64f16870fdf8bd0b81b